.github/workflows/ccpp.yml

```diff
@@ -93,6 +66,38 @@ jobs:
         run: |
           find . -type f \( -name "*.cmake" -o -name "*.cmake.in" -o -name "CMakeLists.txt" \) | xargs cmake-format --check
 
+  build:
+    if: "!(contains(github.event.head_commit.message, '[ci skip]') || contains(github.event.head_commit.message, '[skip ci]'))"
+    runs-on: ubuntu-latest
+    container: ghcr.io/fenics/test-env:current-openmpi
```

Comment:
If you make `build` require `lint` then we can save pointless CI starts across the matrix of petsc architectures.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds